NSelene/Conditions/Not.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace NSelene.Conditions
+{
+    public class Not<TEntity> : DescribedCondition<TEntity>
```

Comment:
pretty straightforward implementation... 
under nselene hood, it's hidden, so if we write something like:

```
S("#foo").Should(Be.Not.Enabled);
```
everything looks awesome, but if somebody created their own condition named for example  `Match.Highlighted`, and don't want to predefine `Match.Not.Highlighted`, but just use:

```
S("#foo").Should(new Not<SeleneElement>(Match.Highlighted));
```

... now it's start to look awkward, bulky, over-verbose...

let's discuss this more in https://github.com/yashaka/NSelene/issues/53